include/procfile.bash

```diff
@@ -18,7 +18,7 @@ procfile-exec() {
 	procfile-load-env
 	procfile-load-profile
 	cd "$app_path"
-	unprivileged /bin/bash -c "$(eval echo $@)"
+	exec setuidgid "$unprivileged_user" $(eval echo $@)
```

Comment:
This seems to make sense. Since `unprivileged` is used more than once, it can't have `exec` because that literally replaces the current process with a new one never to be returned. So I was wrong to suggest `exec` in `unprivileged`. 

That said `exec setuidgid "$unprivileged_user" "$@"` isn't going to evaluate any of the shell related syntax in `$@`. So I think `exec setuidgid "$unprivileged_user" $(eval echo $@)` is the way to go.